packages/grid/x-data-grid/src/hooks/features/columnGrouping/useGridColumnGrouping.ts

```diff
@@ -103,15 +48,6 @@ const createGroupLookup = (columnGroupingModel: GridColumnNode[]): GridColumnGro
 
   return { ...groupLookup };
 };
-export const columnGroupsStateInitializer: GridStateInitializer<
```

Comment:
I may be lost but does this mean that the groups are no longer persisted in the state? Shouldn't we initialize an empty `columnGrouping` state?

Reply:
It's unclear for me when an initializer is necessary. I thaught it was used to initialise the state before pipe processors are called.

Since column grouping is not used by any processors, I removed the initialization. But if it's needed for other purpose, I can bring initializer back

Reply:
Ok, since the state is reused by `columnReorder` and `columnHeader` I put back the initializer